downstream/modules/platform/con-pod-specification-mods.adoc

```diff
@@ -8,13 +8,13 @@ Pods are the equivalent of a machine instance (physical or virtual) to a contain
 Each pod is allocated its own internal IP address, therefore owning its entire port space, and containers within pods can share their local storage and networking.
 
 Pods have a life cycle.
-They are defined, then they are assigned to run on a node, then they run until their container(s) exit or they are removed for some other reason.
-Pods, depending on policy and exit code, may be removed after exiting, or may be retained to enable access to the logs of their containers.
+They are defined, then they are assigned to run on a node, then they run until their containers exit or they are removed for some other reason.
+Pods, depending on policy and exit code, can be removed after exiting, or can be retained to enable access to the logs of their containers.
 
-{PlatformName} provides a simple default Pod specification, however, you can provide a custom YAML, or JSON document that overrides the default Pod specification.
+{PlatformName} provides a simple default Pod specification, however, you can provide a custom YAML, or JSON document that overrides the default pod specification.
```

Comment:
I can't remember the exact rule for pods/Pods but "simple default Pod specification" is uppercase, I presume this is OK?